src/test/java/org/example/AppTest.java

```diff
@@ -31,4 +36,16 @@ public static Test suite() {
     public void testApp() {
         assertTrue( true );
     }
+
+    /**
+     * Test sorting method - sortedList
+     */
+    public void testList(){
```

Comment:
While writing tests please follow this rule https://martinfowler.com/bliki/GivenWhenThen.html. Given/When/Then are just comments

// Given
String data=...
String expected= ...

// When
ArrayList<String> actual = App.sortedList(data);

// Then
Assertions.assertLinesMatch(expected, actual);